.clj-kondo/config.edn

```diff
@@ -15,6 +15,8 @@
   :aliased-namespace-symbol {:level :warning}
   ;; Disable until it doesn't trigger false positives on rum/defcontext
   :earmuffed-var-not-dynamic {:level :off}
+  ;; Disable until we decide to use conj! as recommended in docs
```

Comment:
@RCmerci This linter was warning on a couple uses of conj! so I turned it off. The [general docs/advice](https://clojuredocs.org/clojure.core/conj!) even on mutable fns is to return back the value. Would be nice if you audited these some time and decide whether to change them or ignore the linter. Reviewing them doesn't have to be in this PR